cli/command/stack/kubernetes/loader.go

```diff
@@ -8,16 +8,30 @@ import (
 )
 
 type versionedConfig struct {
-	*composetypes.Config `yaml:",inline"`
-	Version              string
+	composetypes.Config
+	version string
+}
+
+func (c versionedConfig) MarshalYAML() (interface{}, error) {
```

Comment:
I think we'll be able to remove these `MarshalYAML` in the future by making `Config.Services` a type with a `MarshalYAML`.